CMakeLists.txt

```diff
@@ -318,7 +317,9 @@ endif()
 # MEFISTO2
 file(GLOB MEFISTO2_SRC src/MEFISTO2/aptrte.cxx src/MEFISTO2/trte.c)
 add_library(MEFISTO2 ${MEFISTO2_SRC})
-set_target_properties(MEFISTO2 PROPERTIES COMPILE_FLAGS "-DMEFISTO2D_EXPORTS")
+if (WIN32)
```

Comment:
@trelau can you have a look at this? Thiese export definitions are only set for windows for other targets, so it should be the same for this case?

Reply:
Makes sense